Palace.xcodeproj/project.pbxproj

```diff
@@ -4022,7 +4022,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 1.0.15;
+				MARKETING_VERSION = 1.0.16;
```

Comment:
Could you change the version back to 1.0.15? We are going to increase build number onl for Test Flight builds and version number after we publish a new version on App Store.